docs/src/main/sphinx/release/release-353.md

```diff
@@ -0,0 +1,120 @@
+
+## Server RPM
+
+* Allow configuring process environment variables through `/etc/trino/env.sh`. ({issue}`6635`)
+
+## BigQuery connector
+
+* Add support for `CREATE TABLE` and `DROP TABLE` statements. ({issue}`3767`)
+* Allow for case-insensitive identifiers matching via `bigquery.case-insensitive-name-matching` config property. ({issue}`6748`)
+
+## Hive connector
+
+* Add support for `current_user()` in Hive defined views. ({issue}`6720`)
+* Add support for reading and writing column statistics from Glue metastore. ({issue}`6178`)
+* Improve parallelism of bucketed tables inserts. Inserts into bucketed tables can now be parallelized
+  within task using `task.writer-count` feature config. ({issue}`6924`, {issue}`6866`)
+* Fix a failure when `INSERT` writes to a partition created by an earlier `INSERT` statement. ({issue}`6853`)
+* Correctly recognize directory objects created by AWS S3 web console. ({issue}`6992`)
+* Fix query failures on ``information_schema.views`` table when there are failures 
+  translating hive view definitions. ({issue}`6370`)
+
+## Iceberg connector
+
+* Correctly recognize directory objects created by AWS S3 web console. ({issue}`6992`)
+
+## Kafka connector
+
+* Fix failure when querying Schema Registry tables. ({issue}`6902`)
+* Fix querying of Schema Registry tables with References in their schema. ({issue}`6907`)
+* Fix listing of schema registry tables having ambiguous subject name in lower case.
+
+## MySQL connector
+
+* Fix failure when reading a `timestamp` or `datetime` value with more than 3 decimal digits 
+  in the fractional seconds part. ({issue}`6852`)
+* Fix incorrect predicate pushdown for `char` and `varchar` column with operators 
+  like `!=`, `<`, `<=`, `>` and `>=` due different case sensitivity between Trino 
+  and MySQL. ({issue}`6746`, {issue}`6671`)
+
+## MemSQL connector
+
+* Fix failure when reading a `timestamp` or `datetime` value with more than 3 decimal digits 
+  of the second fraction. ({issue}`6852`)
+* Fix incorrect predicate pushdown for `char` and `varchar` column with operators 
+  like `!=`, `<`, `<=`, `>` and `>=` due different case sensitivity between Trino 
+  and MemSQL. ({issue}`6746`, {issue}`6671`)
+
+## Phoenix connector
+
+* Add support for Phoenix 5.1. This can be used by setting `connector.name=phoenix5` in catalog
+  configuration properties. ({issue}`6865`)
+
+## PostgreSQL connector
+
+* Improve performance of queries with `ORDER BY ... LIMIT` clause, when the computation 
+  can be pushed down to the underlying database. This can be enabled by setting `topn-pushdown.enabled`.
+  Enabling this feature can currently result in incorrect query results when sorting
+  on `char` or `varchar` columns. ({issue}`6847`)
+* Fix incorrect predicate pushdown for `char` and `varchar` column with operators 
+  like `!=`, `<`, `<=`, `>` and `>=` due different case collation between Trino 
+  and PostgreSQL. ({issue}`3645`)  
+
+## Redshift connector
```

Comment:
Did we fix predicate pushdown for Redshift too?

Reply:
Also for Oracle?

Reply:
@findepi @kokosing ?

Reply:
for Oracle -- no, since its default behavior is compatible with Trino

for Redshift -- i think we fixed, at least we disabled it. Redshift connector has no tests though, so...